chart/scripts/post-delete-hook.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+set -e
+
+namespaces="${NAMESPACES}"
+timeout="${TIMEOUT}"
+ignoreTimeoutError="${IGNORETIMEOUTERROR}"
+
+if [ -z "${namespaces}" ]; then
+  echo "No namespace is provided."
+  exit 1
+fi
+
+if [ -z "${timeout}" ]; then
+  echo "No timeout value is provided."
+  exit 1
+fi
+
+if [ -z "${ignoreTimeoutError}" ]; then
+  echo "No ignoreTimeoutError value is provided."
+  exit 1
+fi
+
+succeeded=()
+failed=()
+
+get_pod_count() {
+  kubectl get pods --selector app="${1}" -n "${2}" -o json | jq '.items | length'
```

Comment:
Can we assume rancher/shell has jq in it?

Reply:
yes, `jq` is included in the rancher/shell image

https://github.com/rancher/shell/blob/49ecd409727764d202026ab7f28b6820c2e28b5e/package/Dockerfile#L15
```
RUN apk add -U --no-cache bash bash-completion jq
```